src/shared/unit-file.c

```diff
@@ -118,8 +118,33 @@ static int unit_ids_map_get(
         return -ELOOP;
 }
 
+static bool lp_mtime_good(const LookupPaths *lp, usec_t mtime) {
```

Comment:
why abbrev the function name, let's just call this looup_paths_mtime_good?

abbreviation arg names is fine, but function names? just makes it hard to read backtraces